### PR TITLE
Portfolio: Website content

### PR DESCRIPTION
**Pull request by lily4178993**
<h1> Microverse module 1: Week 1</h1>
<h2>Portfolio Project (Mobile version)</h2>

In this branch, I have:

- added a [HTML ](https://github.com/lily4178993/Portfolio/blob/website-content/index.html) and [CSS](https://github.com/lily4178993/Portfolio/blob/website-content/styles.css) files
- added a [gitignore file](https://github.com/lily4178993/Portfolio/blob/website-content/.gitignore)
- added a [README file](https://github.com/lily4178993/Portfolio/blob/website-content/README.md)
- Set up linters for HTML and CSS
- create a [design folder](https://github.com/lily4178993/Portfolio/tree/website-content/design) that contains the screenshots of the mobile version
- create a [styles guideline](https://github.com/lily4178993/Portfolio/blob/website-content/style-guide.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,185 @@
...
+### Run tests
+
+> Coming soon...
+
+<br/><br/>
+
+### <b>Deployment</b>
+
+> Coming soon...
+
```
Comment (tugsem) [inline README.md:120]: - [ ]  Please fill these areas with the information users need to install and run this project locally. If you feel there is nothing to write under a header, kindly remove them.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <!-- Fonts links -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- HEADER -->
+    <header class="header">
+        <div class="logo"><p>My Logo</p></div>
```
Comment (tugsem) [inline index.html:18]: - [ ] Kindly wrap your logo inside an `a` tag. This would be semantically correct.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                <li class="menu-item">Contact</li>
+            </ul>
+        </nav>
+    </header>
+    <!-- HEADER (end) -->
+    <!-- HEADLINE -->
+    <section class="headline">
+        <div class="headline-content">
+            <h1>Hey There.<br>I'm Nelly</h1>
+            <p>I'm a Software Developer</p>
```
Comment (tugsem) [inline index.html:34]: - [ ] This is an example of a subheader. Kindly use `h2` tag to make it semantically correct.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                don’t hesitate to contact me.
+            </p>
+        </div>
+        <div class="socials">
+            <ul>
+                <li><img src="images/github.png" alt="twitter icon"></li>
+                <li><img src="images/Linkedin icon.png" alt="Linkedin icon"></li>
+                <li><img src="images/angel-list.png" alt="twitter icon"></li>
+                <li><img src="images/twitter.png" alt="twitter icon"></li>
+                <li><img src="images/medium.png" alt="twitter icon"></li>
```
Comment (tugsem) [inline index.html:48]: - [ ] Each image in the list item should be wrapped inside an `a` tag. Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @lily4178993 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of github flow
✔️professional commits and pr description
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily4178993** [comment]
Tank you @tugsem for taking your time to review my code 😄 !

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @lily4178993   , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] kindly  Add a descriptive description in your pull request in future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

**Comment by lily4178993** [comment]
Tank you @MaganaAsiati for taking your time to review my code 😄 !

**Pull request merged**